src/Command/CreateReferentTagsForDistrictsCommand.php

```diff
@@ -0,0 +1,63 @@
+<?php
+
+namespace AppBundle\Command;
+
+use AppBundle\Entity\District;
+use AppBundle\Entity\ReferentTag;
+use Doctrine\ORM\EntityManagerInterface;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Style\SymfonyStyle;
+
+class CreateReferentTagsForDistrictsCommand extends Command
+{
+    private $em;
+    private $districtRepository;
```

Comment:
can you add a new line after this property ?

Reply:
Done.